docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
-        </tr>
-        <tr>
-          <th>2</th>
-          <td>120.862054</td>
-          <td>1.630903</td>
-        </tr>
-        <tr>
-          <th>3</th>
-          <td>110.848538</td>
-          <td>2.214022</td>
-        </tr>
-        <tr>
-          <th>4</th>
-          <td>133.763781</td>
-          <td>4.926610</td>
-        </tr>
-        <tr>
-          <th>...</th>
-          <td>...</td>
-          <td>...</td>
-        </tr>
-        <tr>
-          <th>9995</th>
-          <td>135.609528</td>
-          <td>5.082792</td>
-        </tr>
-        <tr>
-          <th>9996</th>
-          <td>91.988864</td>
-          <td>3.282778</td>
-        </tr>
-        <tr>
-          <th>9997</th>
-          <td>90.701909</td>
-          <td>1.307473</td>
-        </tr>
-        <tr>
-          <th>9998</th>
-          <td>122.352818</td>
-          <td>4.338779</td>
-        </tr>
-        <tr>
-          <th>9999</th>
-          <td>136.862764</td>
-          <td>3.626774</td>
-        </tr>
-      </tbody>
-    </table>
-    <p>10000 rows × 2 columns</p>
-    </div>
-
-
-
-.. code:: python
-
-    sns.jointplot(df_synthetic['SBP'], df_synthetic['LDL-C'])
-    rho, p_val = scipy.stats.spearmanr(df_synthetic['SBP'], df_synthetic['LDL-C'])
-    rho
-
-
-
-
-.. parsed-literal::
-
-    0.6797565466113832
-
-
-
-
-.. image:: 2020_02_11a_correlation_and_paf_files/2020_02_11a_correlation_and_paf_13_1.png
-
-
-The anticipated problem with this approach: how does correlation affect the joint PAF?
-======================================================================================
-
-This is not relevant to the material above—using the Gaussian Copula to
-capture the correlation between risk-factor exposure will be fine. The
-problem is when we then layer at RiskEffect component into our model, to
-make the risk have a effect on the incidence or mortality rate of our
-cause model. Without loss of generality, suppose the risk effect will
-change the rate of disease.
-
-We have traditionally handled this by calculating a risk-deleted
-incidence:
-
-.. math::
-
-
-   i_0 = \left(1-\text{PAF}\right) \cdot i_{\text{GBD}}
-
-If we *just* include one risk effect, we should still be ok, even with
-correlation. But if we have two risk effects, we have previously used
-assumption
-
-.. math::
-
-
-   1 - \text{PAF}_{\text{Multiplicative}} = \left(1 - \text{PAF}_{\text{Risk1}}\right)\cdot\left(1 - \text{PAF}_{\text{Risk2}}\right).
-
-The more correlated the risk exposure, the more wrong this assumption.
-
-I propose we continue to assume that the relative risks have a “log
-linear” relationship with incidence:
-
-.. math::
-
-
-   i_{e_1, e_2} = i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2},
+Maintaining the correlation is tricky when the risk exposures for a single simulant change over time.
+In general, we have done this by sampling correlated *percentiles* at initialization and then keeping those *percentiles*
```

Comment:
Again, these are not percentiles but *quantile ranks*.

Reply:
Though I guess maybe it does make sense to say that we keep the same percentiles throughout the simulant's life if we want to be a bit loose with language, e.g.,

"In general, we have done this by sampling correlated *percentile ranks* at initialization and then keeping the simulant's *percentiles* in the exposure distribution static throughout the simulant’s life."